attached-logging-monitoring/monitoring/prometheus.yaml

```diff
@@ -35,7 +38,9 @@ rules:
   - get
   - list
   - watch
+# [END anthos_monitoring_prometheus_clusterrole_stackdriver_user:prometheus]
```

Comment:
are we allowed to have semicolons (`:`) in region tags?

Reply:
whoops!! good catch! 🦅